docs/data/base/getting-started/customization/StylingHooks.tsx

```diff
@@ -69,13 +69,13 @@ function BasicSwitch(props: UseSwitchParameters) {
   };
 
   return (
-    <BasicSwitchRoot className={clsx(stateClasses)}>
-      <BasicSwitchThumb className={clsx(stateClasses)} />
-      <BasicSwitchInput {...getInputProps()} aria-label="Demo switch" />
-    </BasicSwitchRoot>
+    <SwitchRoot className={clsx(stateClasses)}>
+      <SwitchThumb className={clsx(stateClasses)} />
+      <SwitchInput {...getInputProps()} aria-label={props['aria-label']} />
+    </SwitchRoot>
   );
 }
 
 export default function StylingHooks() {
-  return <BasicSwitch />;
+  return <Switch aria-label="Demo switch" />;
 }
```

Comment:
The custom aria-label value inside the `Switch` component didn't make sense to me. This is can't be part of the design system.

Actually, this makes me wonder why in Base UI switch we aren't forwarding all the props to the hook. I find this strange:

https://github.com/mui/material-ui/blob/5c96f3cad765c259d029321a8f7d2f57c9bcc36b/packages/mui-base/src/Switch/Switch.tsx#L69-L80

It could be this, no?

```jsx
  const { getInputProps, checked, disabled, focusVisible, readOnly } = useSwitch(props);
```

Reply:
Yeah, good point :+1: